opm-core/Backups.rst

```diff
@@ -0,0 +1,22 @@
+Backups
+=======
+
+Usual backup procedures apply. The procedure should include the way to reinstall the extensions on a virgin server.
```

Comment:
I'm not sure that I understand what this is explaining.

Reply:
Dropped. Replaced by " use pg_dump as usual."